cloudendure_demo/playbooks/hosts

```diff
@@ -0,0 +1,10 @@
+[all:vars]
+ansible_user=centos 
+ansible_ssh_private_key_file=(ssh_key_pair_path)
+gogs_db_pass=hv123!ZPPos!
```

Comment:
@CarmenAPuccio While this is just an example password, what is the risk if a customer uses the demo this as-is?